tools/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Azure SDK Tools Contribution Guidelines
+
+This is the repository of tool libraries used by Azure SDK purpose.
+For every newly developing tool, create a new folder with name of brief description of the tool. 
```

Comment:
I would suggest putting this under a new markdown header/section like **Adding a new tool**.